### ISSUE TITLE
How does it work?

### ISSUE THREAD

**Issue by betrayerofmemes** [twitchdev/stardew-valley-extension#13]
Installing the mod and the extention doesn't do anything. Is there a guide how to propertly do it?

**Comment by 21Z** [comment]
the guy who opened this issue (@betrayerofmemes) is already dead. bcuz on apr 10 he somehow managed to get this to work! and the moment it worked sm1 rang his doorbell and a guy came and fucked him and he died of fucking so hard. gay.

**Comment by betrayerofmemes** [comment]
> the guy who opened this issue (@betrayerofmemes) is already dead. bcuz on apr 10 he somehow managed to get this to work! and the moment it worked sm1 rang his doorbell and a guy came and fucked him and he died of fucking so hard. gay.

wrong. corrong. i still didn't manage to get this to work and still waiting the solution. halp ples

**Comment by 21Z** [comment]
wow sm1 revived u
